config/settings/base.py

```diff
@@ -213,7 +213,21 @@
 # https://docs.celeryproject.org/en/stable/userguide/optimizing.html#broker-connection-pools
 # https://docs.celeryq.dev/en/latest/userguide/optimizing.html#broker-connection-pools
 CELERY_BROKER_POOL_LIMIT = env(
-    "CELERY_BROKER_POOL_LIMIT", default=env("CELERYD_CONCURRENCY", default=1000)
+    "CELERY_BROKER_POOL_LIMIT", default=env("CELERYD_CONCURRENCY", default=0)
```

Comment:
@Uxio0 I changed this value to 0 to open and close connections every time it is needed, maybe could be better a value less than 1000. I'm not sure about to put this to 0, because connections with rabbitMq were designed to be durable.

Reply:
This is not related to connections being durable, only to the number of connections to allow at the same time